src/up.cmd.js

```diff
@@ -26,6 +26,7 @@ export default class UpCommand extends AbstractServerCommand {
 
   withUrl(value) {
     this._url = value;
+    this._pagesUrl = value;
```

Comment:
I would name this `_originalUrl` and remove the `_url` assignement.
or kee `._url` and instead use a `_.sanitizedUrl` after it is computed